integration-tests/src/main/java/org/apache/polaris/service/it/env/CatalogApi.java

```diff
@@ -149,6 +149,18 @@ public void dropTable(String catalog, TableIdentifier id) {
     }
   }
 
+  public int loadTable(String catalog, TableIdentifier id, String snapshots) {
+    String ns = RESTUtil.encodeNamespace(id.namespace());
+    try (Response res =
+        request(
+                "v1/{cat}/namespaces/" + ns + "/tables/{table}",
+                Map.of("cat", catalog, "table", id.name()),
+                Map.of("snapshots", snapshots))
+            .get()) {
+      return res.getStatus();
+    }
+  }
+
```

Comment:
Wouldn't it be nicer if this method actually returned the `LoadTableResponse` object? This way your test could inspect the returned snapshots.

```suggestion
  public LoadTableResponse loadTable(String catalog, TableIdentifier id, String snapshots) {
    String ns = RESTUtil.encodeNamespace(id.namespace());
    try (Response res =
        request(
                "v1/{cat}/namespaces/" + ns + "/tables/{table}",
                Map.of("cat", catalog, "table", id.name()),
                snapshots == null ? Map.of() : Map.of("snapshots", snapshots))
            .get()) {
      if (res.getStatus() == Response.Status.OK.getStatusCode()) {
        return res.readEntity(LoadTableResponse.class);
      }
      throw new RESTException(
          "Unhandled error: %s",
          ((ErrorHandler) ErrorHandlers.defaultErrorHandler())
              .parseResponse(res.getStatus(), res.readEntity(String.class)));
    }
  }
  
```